packages/core/src/index.ts

```diff
@@ -51,6 +51,10 @@ export type BaseGraphQLServerOptions<TContext> = {
    * Context
    */
   context?: (req: Request) => Promise<TContext> | Promise<TContext>
+  /**
+   * Provide scalar for File and Blob
+   */
+  enableUploads?: boolean
```

Comment:
Not sure about the name of that flag because it doesn't actually enable or disable uploads. You can still use upload feature with dummy scalars, or include scalars on your own.

Reply:
True 🤔 what do you think of `injectFileBlobScalar`?